clang/lib/DPCT/Diagnostics.cpp

```diff
@@ -48,7 +48,25 @@ std::unordered_map<int, DiagnosticsMessage> CommentIDTable;
 #undef DEF_WARNING
 #undef DEF_COMMENT
 
-std::unordered_set<int> APIQueryNeedReportWarningIDSet = {1086};
+std::unordered_set<int> APIQueryNeedReportWarningIDSet = {
+    1007, // API_NOT_MIGRATED
```

Comment:
Let's remove 1007, it is confusing that dpct give out a mapping result, but at same time told 1007.

Reply:
fixed